__tests__/components/ResourceCounts.test.tsx

```diff
@@ -34,12 +30,20 @@ describe("resource Counts render", () => {
     expect(screen.getByRole("link", { name: "Appointment 0" })).toBeInTheDocument();
   });
 
-  it("the retrieved resources should be sorted by count, then alphabetically", async () => {
```

Comment:
We got rid of this test that checks that resources are sorted by count, then alphabetically. We can and probably should still test this, either with the whole list or with a search filtered list or both.

Reply:
I talked to Matt about it before removing it and he said not to test this function individually. Because it relies on state variables, we don't want to separate it from its component to export it and we couldn't think of a way to test this through the component unit tests. Happy to add it back though if you both think it's necessary!